tcms/testcases/models.py

```diff
@@ -418,14 +418,10 @@ def add_text(
         return latest_text
 
     def add_to_plan(self, plan):
-
-        try:
-            TestCasePlan.objects.get(case=self, plan=plan)
-        except TestCasePlan.DoesNotExist:
-            TestCasePlan.objects.get_or_create(
-                case=self,
-                plan=plan,
-            )
+        TestCasePlan.objects.get_or_create(
+            case=self,
+            plan=plan,
+        )
```

Comment:
Thought about code style of ORM. For this simple case, would this be more compact

```
TestCasePlan.objects.get_or_create(case=self, plan=plan)
```